pandas/_libs/src/klib/khash_python.h

```diff
@@ -330,6 +333,40 @@ Py_hash_t PANDAS_INLINE tupleobject_hash(PyTupleObject* key) {
 }
 
 
+// TODO: same thing for timedelta64 objects
+Py_hash_t PANDAS_INLINE np_datetime64_object_hash(PyObject* key) {
```

Comment:
I don't think this function is inlinable at all - does this do anything?

Reply:
no idea, copy-pasted.  will remove